drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
```

Comment:
@aboch why did we need this change? This will require a lot of change in the docker side because we are already passing all the maps as corresponding type. So why would `GenerateFromModel` not solve a value which is a string since `map[string]string` is just a special case of `map[string]interface{}`